samples/Sentry.Samples.Maui/MauiProgram.cs

```diff
@@ -53,6 +54,10 @@ public static MauiApp CreateMauiApp()
                 fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
             });
 
+        // Automatically create traces for async relay commands
+        // in the MVVM Community Toolkit
+        builder.Services.UseSentryCommunityToolkitIntegration();
```

Comment:
TLDR; can we make `UseSentryCommunityToolkitIntegration` an extension method on `SentryMauiOptions` instead of `IServiceCollection`?

All of our other integrations get initialised via the SentryOptions when initialising the Sentry SDK. In this case, maybe it's not an `ISdkIntegration` like those but it'd be nice if it was done via the `SentryMauiOptions` in the call to `UseSentry`.

We already [register a bunch of others](https://github.com/getsentry/sentry-dotnet/blob/08a82bc9490e2e2337e2815f8f80124f968a2dbc/src/Sentry.Maui/SentryMauiAppBuilderExtensions.cs#L58-L62) in that method...

Reply:
I need dependency injection and IServiceCollection.  SentryMauiOptions does not have that.  You really need a builder pattern if you want to create plugins like that.

Reply:
How about if the call to `UseSentryCommunityToolkitIntegration` stores something in on internal property like `SentryMauiOptions.CustomEventBinders` or something like that. 

Then the call to `UseSentry`, in addition to explicitly registering the event binders it already registers, also checks if there are any `CustomEventBinders` in the options and registers those as well.

That should work right?